files/scripts/swss.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
 
-DEPENDENT="radv dhcp_relay"
+DEPENDENT="radv"
 MULTI_INST_DEPENDENT="teamd"
 
+# Update dependent list based on other packages requirements
```

Comment:
is this change related to this dhcp_relay PR? It seems to me it should apply to every feature that is moving to appxt.